extras/rya.export/README.md

```diff
@@ -0,0 +1,250 @@
+specific language governing permissions and limitations
+under the License. -->
+
+# Rya Export
+
+___
+
+## Purpose
+
+Rya Export is a java application that can be used to export from a parent Rya datastore to a child Rya datastore. The changes to the child can then be synced back to the parent when the process is run again after the child datastore has collected/modified its data set. The datastores currently supported are Accumulo and MongoDB.
+
+![Figure 1: Child to Parent Merge](resources/images/child_to_parent_merge.png)
+
+## Overview
+
+At deployment time, a Rya child instance will be created and sent out where its users will create, modify, and delete data. This data from the child will then need to be synced with its parent after deployment. By default, the data only needs to be handled at the datastore level and so will not need to know the logical consistency of the higher level models it's storing (i.e. Rya logic). Some decisions that were made about the tool:
+
+  * When copying, the parent is copying its data to its child and can optionally filter its data to be after a specified timestamp.
+
+  * When merging, the data is going only ONE way: the child is re-integrating its new data with the parent Rya instance (the parent does NOT have to merge its new data into the child).
+
+  * For Accumulo datastores, since an update to a row value is just a delete followed by an add, row value updates aren't needed.
+
+  * For Accumulo datastores, special care must be taken when there's a difference between the parent's and child's column visibility in the same row. In this case, the two visibility values should be OR'ed together to produce the new visibility value to use.
+  
+  * For Accumulo to MongoDB merging, column visibility is ignored when updating.
+
+  * The tool should support merging data from after a specific time (start time). This will merge in a smaller subset of data modifications from the child to the parent.
+  
+  * The Rya Export Client can handle copying/merging from Accumulo to Accumulo, Accumulo to MongoDB, MongoDB to Accumulo, and MongoDB to MongoDB.
+
+## Rya Export Client Process
+
+The Rya Export Client can be ran through the command line. It can either be used to copy a child instance from a parent or it can be used to sync an already exported child instance by merging its data over to the parent instance. The user can configure a timestamp for the data to be included so only data coming after that time is replicated. The command line will make calls to an uber jar containing the tool and execute the job. This will also allow extra config parameters (such as the location of the parent and child instance) to be used when called.
+
+## Implementation Details
+
+The Rya Export Client can be run on any machine connected to both the parent instance machine and the child instance machine. It determines which machine is the parent and child by storing metadata that marks the parent machine after the initial copy.
```

Comment:
This is ambiguous, The parent metadata is stored on the child, this makes it seem like the parent could have the metadata stored on it